### ISSUE TITLE
Can't modify storage policy at container and object level with the CLI

### ISSUE THREAD

**Issue by GuillaumeDelaporte** [open-io/oio-sds#813]
Looking at the code of the CLI it seems that we can modify the storage policy of a container: https://github.com/open-io/oio-sds/blob/master/oio/cli/storage/container.py#L27

But while trying to use both `--storage-policy` and `--stgpol` I faced an `unrecognised argument`:
```
openio object create: error: unrecognized arguments: --storage-policy=ERASURECODE
```

I faced the same issue at the object level, but after many tries with @racciari we found that we can do it only at the object level with `openio object create mycontainer2 /etc/hosts --policy=ERASURECODE`.

Can you make the storage policy modification available for the CLI, as well as modifying the usage and the documentation?

Cheers and Merry Christmas!

**Comment by jkasarherou** [comment]
```
 openio container create 3 --storage-policy=THREECOPIES
+------+---------+
| Name | Created |
+------+---------+
| 3    | True    |
+------+---------+
```

```
 openio object create 1 /etc/fstab  --policy THREECOPIES
+-------+------+----------------------------------+
| Name  | Size | Hash                             |
+-------+------+----------------------------------+
| fstab |  620 | CB8165534EE8F2D261F34CAAE6448D63 |
+-------+------+----------------------------------+
```
What is the issue here?

**Comment by GuillaumeDelaporte** [comment]
The documentation then 😃 

By the way, the problem was fixed (and the usage was updated), with the last packages published. I faced the issue in 3.0.0, now fixed in 3.1.2.
I close the issue.

Thanks.
